Makefile

```diff
@@ -84,6 +84,18 @@ revive: scripts/go/bin/revive
 		-config ./scripts/go/configs/revive.toml \
 		$(GO_FILES)
 
+# create docker-compose file with provided sources and start them
+env: env-down
```

Comment:
How is this supposed to be used? Not sure if `env actually need to depend on `env-down`, `docker-compose up` should take care of removing cointainers as well as adding them unless I'm mistaken.

Reply:
Added an example as makefile comment  – 

`$ make env sources=postgres,openldap`

> Not sure if env actually need to depend on env-down, docker-compose up` should take care of removing cointainers as well as adding them unless I'm mistaken.

Just tried it - If you execute `docker-compose up` its enters the shell of the containers defined in `docker-compose.yaml` it does not extinguish already running containers that *were* defined in `docker-compose.yaml` but not anymore

Reply:
`docker-compose down` solved broken docker-compose setup for me a bunch of times. I would not mind depending on it to reduce errors on `up`